File: data/c#/pattern_26/code.cs
using System;
class Grammind
{
    static void Main()
    {
        char ch;
        Console.Write("Enter the number of rows and columns for the square: ");
        ch = Char.Parse(Console.ReadLine());

        for (char i = 'A'; i <= Char.ToUpper(ch); i++)
        {
            for (char j = 'A'; j <= Char.ToUpper(ch); j++)
            {
                Console.Write("{0,2:D }", i);
            }
            Console.Write("\n");
        }
    }
}

File: data/c/pattern_26/code.c
#include <ctype.h>
#include <stdio.h>

int main(void)
{
    char ch;
    printf("Enter the character of rows and columns for the square: ");
    scanf("%c", &ch);

    for (char i = 'A'; i <= toupper(ch); i++)
    {
        for (char j = 'A'; j <= toupper(ch); j++)
        {
            printf("%c ", i);
        }
        printf("\n");
    }

    return 0;
}
